dbt/models/output/out_eia__monthly_generators/schema.yml

```diff
@@ -36,12 +36,7 @@ sources:
                   weight_column: capacity_mw
               - expect_column_weighted_quantile_values_to_be_between:
                   quantile: 0.65
-                  max_value: |
-                    {% if target.name == "etl-fast" %}
-                    0.9
-                    {% else %}
-                    0.7
-                    {% endif %}
+                  max_value: 0.7
```

Comment:
I expect that we'll see a bunch of errors if the tests are run against the fast ETL outputs -- that will probably be tthe hard part about migrating these tests.